mypy/test/testcmdline.py

```diff
@@ -49,8 +50,11 @@ def test_python_cmdline(testcase: DataDrivenTestCase, step: int) -> None:
             file.write('{}\n'.format(s))
     args = parse_args(testcase.input[0])
     custom_cwd = parse_cwd(testcase.input[1]) if len(testcase.input) > 1 else None
+    if "# dont-normalize-output:" in testcase.input:
```

Comment:
what that means

Reply:
```
[case testSomething]
from typing import Literal
a: Literal["\\"] 
reveal_type(a)  # N: Revealed type is "Literal['\']"
```
This will fail with something like:
```
expected: 
note: Revealed type is "Literal['\']"
actual
note: Revealed type is "Literal['/']"
```
Because the test framework normalizes the output so that tests that output path separators work on windows.